wgsl/index.bs

```diff
@@ -1620,8 +1624,8 @@ TODO: Type rules for vector access
   <thead>
     <tr><td>Precondition<td>Conclusion<td>Notes
   </thead>
-  <tr><td>*e* : *T*, *T* is *Integral*<td>`-e` : *T*<td>OpSNegate
-  <tr><td>*e* : *T*, *T* is *Floating*<td>`-e` : *T*<td>OpFNegate
+  <tr><td>*e* : *T*, *T* is *SignedIntegral*<td>`-e` : *T*<td>Signed integer negation. OpSNegate
```

Comment:
Editorial note:  We could expand the descriptions of the many of the operators already in the draft.
I'll leave that to future MRs.